src/libtomahawk/playlist/TrackDetailView.cpp

```diff
@@ -441,3 +441,11 @@ TrackDetailView::setBuyButtonVisible( bool visible )
 {
     m_buyButtonVisible = visible;
 }
+
+
+void
+TrackDetailView::onResultClicked( const Tomahawk::playlistinterface_ptr& playlist, const Tomahawk::result_ptr& result, const Tomahawk::query_ptr& fromQuery )
+{
+    fromQuery->setPreferredResult( result );
```

Comment:
Do you think it would be better to *just* move the result, ie not play item right away?